flask_openapi3/scaffold.py

```diff
@@ -89,6 +95,18 @@ async def view_func(**kwargs) -> FlaskResponse:
                     response = await func(view_object, **func_kwargs)
                 else:
                     response = await func(**func_kwargs)
+
+                if hasattr(current_app, "validate_response"):
+                    if validate_response is None:
+                        _validate_response = current_app.validate_response
+                    else:
+                        _validate_response = validate_response
+                else:
+                    _validate_response = validate_response
+
+                if _validate_response and responses:
+                    run_validate_response(response, responses)
```

Comment:
I would prefer that run_validate_response() be able to modify the response and then you return it. What do you think?